bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/igrins2/Igrins2.scala

```diff
@@ -54,6 +54,8 @@ case class Igrins2(blueprint: SpIgrins2Blueprint) extends GroupInitializer[SpIgr
   //
   //  INCLUDE the note "Observer Instructions" IN the target-specific Scheduling Group
 
+  addNote("Phase II Checklist") in TopLevel
```

Comment:
This note is supposed to go at the top level.  Unfortunately it appears that we ignore the `NoteLocation` in the template DSL.  You can see [here](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/TemplateDsl2.scala#L39-L44) that the `notes` definition throws away the `NoteLocation`.  I'm loathe to try and fix this.

Reply:
SortedMap?

Reply:
I'm going to see if I can get away with this.  There are other notes that are supposed to be at the `TopLevel` but aren't.

Reply:
Oh, it looks like [this is expected](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/TemplateFactoryImpl.scala#L164-L169).

```scala
    // Add all notes to target group for now
    val p = db.progMap(ini.program)
    ini.notes.flatMap(findNote(p)(_).toList).foreach { n =>
      val n0 = db.odb.getFactory.createObsComponentCopy(grp.getProgram, n, false)
      grp.addObsComponent(n0)
    }
```